tool/client.cc

```diff
@@ -378,18 +453,86 @@ static void InfoCallback(const SSL *ssl, int type, int value) {
   }
 }
 
-bool Client(const std::vector<std::string> &args) {
-  if (!InitSocketLibrary()) {
-    return false;
+static int verify_cb(int ok, X509_STORE_CTX *ctx)
+{
+  X509 *err_cert;
+  int err, depth;
+  BIO* bio_err = BIO_new_fp(stdout, BIO_CLOSE);
+
+  err_cert = X509_STORE_CTX_get_current_cert(ctx);
+  err = X509_STORE_CTX_get_error(ctx);
+  depth = X509_STORE_CTX_get_error_depth(ctx);
```

Comment:
Minor nit: we can create and directly assign the variables in the same line here